README.md

```diff
@@ -1,6 +1,6 @@
 # 🚩 Challenge #0: 🎟 Simple NFT Example - SpeedRunEthereum
 
-🎫 Create a simple NFT to learn basics of 🏗 scaffold-eth. You'll use [👷‍♀️ HardHat](https://hardhat.org/getting-started/) to compile and deploy smart contracts. Then, you'll use a template React app full of important Ethereum components and hooks. Finally, you'll deploy an NFT to a public network to share with friends! 🚀
+🎫 Create a simple NFT to learn basics of 🏗 Scaffold-ETH 2. You'll use [👷‍♀️ HardHat](https://hardhat.org/getting-started/) to compile and deploy smart contracts. Then, you'll use a template React app full of important Ethereum components and hooks. Finally, you'll deploy an NFT to a public network to share with friends! 🚀
```

Comment:
How about adding this version of the image here?

![photo1687360630](https://github.com/scaffold-eth/se-2-challenges/assets/2486142/3f1dcefc-7c1a-4a43-9857-368059d735b6)